src/nvim/testdir/test_autocmd.vim

```diff
@@ -816,8 +816,8 @@ func Test_Cmdline()
   new
   call setline(1, 'hello')
   call feedkeys("/hello\<CR>", 'xt')
-  call assert_equal('/', g:entered)
-  call assert_equal('/', g:left)
+  call assert_equal(has('win32') ? 'C:/' : '/', g:entered)
+  call assert_equal(has('win32') ? 'C:/' : '/', g:left)
```

Comment:
This isn't needed in upstream Vim: https://github.com/vim/vim/blob/e7499ddc33508d3d341e96f84a0e7b95b2d6927c/src/testdir/test_autocmd.vim#L816

When changing the oldtests, we need to be careful not to add a bunch of tiny changes except where absolutely unavoidable. Otherwise it makes merging pain in the ass.

And we need to ask why the result is different. Is this an intentional behavior change in `expand('<afile>')` that we plan to keep? If not, there's no reason to make the test pass.

Reply:
Are they the same internally but Vim hides the network drive for `<afile>`?